File: Algorithms/p242_Valid Anagram_2.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        cnt1 = [0]*256
        cnt2 = [0]*256
        for c in s:
            cnt1[ord(c)] += 1
        for c in t:
            cnt2[ord(c)] += 1
        return cnt1==cnt2

File: Algorithms/p242_Valid Anagram_2.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int cnt1[256] = {};
        int cnt2[256] = {};
        for(const char& c : s){
            cnt1[c]++;
        }
        for(const char& c : t){
            cnt2[c]++;
        }
        for(int i=0;i<256;i++){
            if(cnt1[i]!=cnt2[i]){
                return 0;
            }
        }
        return 1;
    }
};
